src/test/java/org/assertj/core/internal/int2darrays/Int2DArrays_assertDoesNotContain_at_Index_Test.java

```diff
@@ -0,0 +1,79 @@
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.internal.int2darrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.Assertions.assertThatNullPointerException;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.assertj.core.data.Index.atIndex;
+import static org.assertj.core.error.ShouldNotContainAtIndex.shouldNotContainAtIndex;
+import static org.assertj.core.test.TestData.someIndex;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+import static org.mockito.Mockito.verify;
+
+import org.assertj.core.api.AssertionInfo;
+import org.assertj.core.data.Index;
+import org.assertj.core.internal.Int2DArrays;
+import org.assertj.core.internal.Int2DArraysBaseTest;
+import org.junit.jupiter.api.Test;
+
+
+/**
+ * Tests for <code>{@link Int2DArrays#assertDoesNotContain(AssertionInfo, int[][], int[], Index)}</code>.
+ * 
+ * @author Alex Ruiz
+ * @author Joel Costigliola
+ * @author Maciej Wajcht
+ */
+public class Int2DArrays_assertDoesNotContain_at_Index_Test extends Int2DArraysBaseTest {
+
+  @Test
+  public void should_fail_if_actual_is_null() {
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> arrays.assertDoesNotContain(someInfo(), null, new int[] {0, 2, 4}, someIndex()))
+                                                   .withMessage(actualIsNull());
+  }
+
+  @Test
+  public void should_pass_if_actual_does_not_contain_value_at_Index() {
+    arrays.assertDoesNotContain(someInfo(), actual, new int[] {0, 2, 4}, atIndex(1));
+  }
+
+  @Test
+  public void should_pass_if_actual_is_empty() {
+    arrays.assertDoesNotContain(someInfo(), new int[][] {}, new int[] {0, 2, 4}, someIndex());
+  }
+
+  @Test
+  public void should_throw_error_if_Index_is_null() {
+    assertThatNullPointerException().isThrownBy(() -> arrays.assertDoesNotContain(someInfo(), actual, new int[] {0, 2, 4}, null))
+                                    .withMessage("Index should not be null");
+  }
+
+  @Test
+  public void should_pass_if_Index_is_out_of_bounds() {
+    arrays.assertDoesNotContain(someInfo(), actual, new int[] {0, 2, 4}, atIndex(6));
+  }
+
+  @Test
+  public void should_fail_if_actual_contains_value_at_index() {
+    AssertionInfo info = someInfo();
+    Index index = atIndex(0);
+
+    Throwable error = catchThrowable(() -> arrays.assertDoesNotContain(info, actual, new int[] {0, 2, 4}, index));
```

Comment:
use `expectAssertionErrror` in place of `catchThrowable`, no need then to get the assertion error, `verify(failures)...`  is enough since `expectAssertionErrror` will check that an AssertionError was thrown

Reply:
done